packages/tooling/fast-tooling/src/web-components/css-layout/css-layout.ts

```diff
@@ -1,4 +1,5 @@
 import { observable } from "@microsoft/fast-element";
+import { keyCodeSpace } from "@microsoft/fast-web-utilities";
```

Comment:
Nit: should move away from keyCodes.

Reply:
Yep, there's no utility to replace it yet though.